runtime/onert/backend/train/ops/MeanLayer.cc

```diff
@@ -48,26 +48,11 @@ void MeanLayer::forward(bool) { cpu::ops::MeanLayer::run(); }
 
 void MeanLayer::backward()
 {
-  nnfw::cker::Shape keep_dim_shape;
-  if (_keep_dims == false)
-  {
-    keep_dim_shape.ReplaceWith(getShape(_input));
-    auto axes_vec = cpu::ops::getReducerAxes(_axes);
-    for (const auto &axis : axes_vec)
-    {
-      keep_dim_shape.SetDim(axis, 1);
-    }
-  }
-  else
-  {
-    keep_dim_shape.ReplaceWith(getShape(_back_prop_input));
-  }
-
```

Comment:
I think this code can be removed if there is no dynamic tensor.

Reply:
It looks okay. Is it possible to add assert it is not dynamic tensor before removing this code?

Reply:
Sure. I will add it in this PR.

Reply:
I create #13358 to add the assert.